app/models/lockstep/ar_header.rb

```diff
@@ -0,0 +1,28 @@
+class Lockstep::ArHeader < Lockstep::ApiRecord
+  self.model_name_uri = "v1/Reports/ar-header"
+  self.id_ref = "group_key"
+  self.query_path= ""
+  load_schema(Schema::ArHeaderInfo)
+
+end
+
+class Lockstep::Query
+  def build_params
+    params = {}
+    params.merge!({ :reportDate => criteria[:conditions].first[:report_date]})
```

Comment:
conditions can be blank right that time .first would give a nil 🤔  can we have this code a bit more generic , like loop through conditions array and then based on key merge it to params and not hardcode :report_date key

Reply:
Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/14/commits/cad33e04ad2a1cb91088de1741978cbdad1ae26e#diff-d30637425d8720912902815c0427d6111a14c7a9eff8a31bb1cc29da3706c8bfR12)